crates/core/tedge_api/src/mqtt_topics.rs

```diff
@@ -180,6 +180,7 @@ impl MqttSchema {
             ChannelFilter::AnyCommandMetadata => "/cmd/+".to_string(),
             ChannelFilter::CommandMetadata(operation) => format!("/cmd/{operation}"),
             ChannelFilter::Health => "/status/health".to_string(),
+            ChannelFilter::AnyData => "/+/#".to_string(),
```

Comment:
Instead of this broad topic filter that covers "everything", we can selectively subscribe to the known channels like [`/a/+/meta`, `/cmd/+`, `/cmd/+/+`, ...] as well.

Reply:
Resolved by 67a05b3c